platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/server/IdpEndpoint.java

```diff
@@ -161,6 +110,14 @@
 
     private Boolean strictSignature = true;
 
+    private SystemCrypto systemCrypto;
+
+    public IdpEndpoint(String signaturePropertiesPath, String encryptionPropertiesPath,
```

Comment:
@codice/security pretty interesting Coverity finding here. As the SystemCrypto constructor has the arguments in the opposite order with encryptionPropertiesPath first, then signaturePropertiesPath. In the IdpEndpoint we currently have that order reversed. https://scan8.coverity.com/reports.htm#v25664/p10941/fileInstanceId=9575771&defectInstanceId=3337657&mergedDefectId=132822